packages/fxa-email-service/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.135.2
```

Comment:
Do you think it is worthwhile to not add entries here if there are not changes?

Reply:
That would break the release script as it stands, because it looks for the preceding version number to decide where it should insert the change summary. Although of course, if we want to change how that part of the script works then anything is possible.